elasticdl/python/tests/model_handler_test.py

```diff
@@ -178,22 +200,40 @@ def test_get_model_with_embedding_layer_to_train(self):
     def test_get_model_with_embedding_column_to_train(self):
         model_inst = custom_model_with_embedding_column()
         self.assertEqual(
-            type(model_inst.layers[0]._feature_columns[1]),
+            type(model_inst.layers[2]._feature_columns[1]),
```

Comment:
Assign 2 to a constant and give it a meaningful name so the code is more readable